### DISCUSSION TITLE
Change or Apply Color Scheme in Access Database

### DISCUSSION THREAD

**Discussion by abidkhanpk** [joyfullservice/msaccess-vcs-addin#339] [Q&A] [answered]
Hello,

This question is not directly related to VCS plugin. But I think it can help me solve my issue which I have been trying to solve for many days. This plugin can export color scheme of access database and import and reapply it on database build. I want to know the code that can do that. The purpose is, I want to put functionality of changing color schemes in my database but couldn't find a way to apply a thmx in runtime. Can somebody tell me how this plugin handles it.

Regards

**Comment by joyfullservice** [comment, marked as answer]
As an open source project, the VBA source code is all available for you to review.  😄 Most of the theme related code is going to be in the VBA class `clsDbTheme`. Themes are stored in the `MSysResources` table as embedded file attachments. There is also a database property that stores the current theme name.

Hope that gives you some good clues to get started...  😄

**Reply by abidkhanpk** [reply]
Thanks for reply. I will look into the class module and will update if I make any progress in my issue. Thanks again.
